projects/spectator/src/lib/spectator-routing/router-link-stub.ts

```diff
@@ -14,15 +14,32 @@ import { RouterLink } from '@angular/router';
 })
 export class RouterLinkDirectiveStub {
```

Comment:
Thanks for the PR. Now that I'm thinking about it, I can't remember why we have a stub here. Can you try to use the original directive, please?

Reply:
from what I can gather, it looks like I can't just do `moduleMetadata.declarations.push(RouterLink);` instead of `moduleMetadata.declarations.push(RouterLinkStub);`

since angular 9, you need to import the module of the library directly
https://github.com/angular/angular/issues/37047#issuecomment-634226777

so turns out I need to import `RouterTestingModule` along with the other stubs when they are enabled

I was able to make it work but I needed to mock the serializeUrl function in the routerStub, as angular was trying to parse the url using it internally
```
serializeUrl(): string {
  return '/';
},
```

Reply:
I'm actually not sure if the routerStub is still needed since I'm now importing `RouterTestingModule`, it looks like it should really just stub `ActiveRoute`